### PR TITLE
flip-filter and socket-connect

### LINKED ISSUE #2: Implement filter functionality for flipper
Implement a filter functionality for the flipper

### PR DESCRIPTION
**Pull request by Ekwav**
fixes #2

### CODE REVIEW

File: Models/FlipSetings.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
+
```
Comment (matthias-luger) [inline Models/FlipSetings.cs:1]: Typo in filename -> FlipSettings.cs

File: Models/FlipSetings.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using hypixel;
+
+namespace Coflnet.Sky.Filter
+{
+
+    [DataContract]
+    public class FlipSettings
```
Comment (matthias-luger) [inline Models/FlipSetings.cs:11]: Why do you not accept maxCost?

File: Services/FlipperService.cs [hunk 3]
```diff
@@ -184,7 +198,8 @@ public async Task ProcessSlowQueue()
                     if (SoldAuctions.ContainsKey(flip.UId))
                         flip.Sold = true;
                     NotifyAll(flip, SlowSubs);
-                    Console.Write("sf+" + SlowSubs.Count);
+                    if (flip.Uuid[0] == 'a')
```
Comment (matthias-luger) [inline Services/FlipperService.cs:201]: Why do you log all flips with a UUID starting with 'a'?
Only a notice, as I dont care what you log.

File: Socket/IFlipConection.cs [hunk 4]
```diff
@@ -1,9 +1,20 @@
+using Coflnet.Sky.Filter;
```
Comment (matthias-luger) [inline Socket/IFlipConection.cs:1]: Typo in filename -> IFlipConnection.cs

**Review by matthias-luger — CHANGES_REQUESTED** [review]

File: Services/FlipperService.cs (hunk 3 above)
Comment (Ekwav) [reply Services/FlipperService.cs:201]: This is just there to see if there still flips sent.
Its effectively just logging 1/16th of the flips to reduce the amount of logs that are stored

**Review by matthias-luger — APPROVED** [review]

**Pull request merged**